### PR TITLE
ML-028/add policies

### PR DESCRIPTION
**Pull request by WhatWasThatt**
https://trello.com/c/xG3do7jZ

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -11,6 +11,9 @@
       resources :roles
       resources :permissions
       resources :tags
+      resources :user
+      put 'lock_user', to: 'user#lock_user'
```
Comment (dmitrymalllpr) [inline config/routes.rb:15]: devise_for ???

**Review by dmitrymalllpr — CHANGES_REQUESTED** [review]

File: config/routes.rb (hunk 1 above)
Comment (dmitrymalllpr) [reply config/routes.rb:15]: Please use device routes

File: app/policies/person_policy.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+class PersonPolicy < BasePolicy
+end
```
Comment (dmitrymalllpr) [inline app/policies/person_policy.rb:2]: strange... Role is not Policy. Better EntityPolicy<BasePolicy   DictionaryEntityPolicy <DictionaryPolicy ect

**Review by KarinaKlevzhits — APPROVED** [review]

**Review by dmitrymalllpr — CHANGES_REQUESTED** [review]

File: app/policies/person_policy.rb (hunk 2 above)
Comment (WhatWasThatt) [reply app/policies/person_policy.rb:2]: should I remove the inheritance from permission?

Comment (dmitrymalllpr) [reply app/policies/person_policy.rb:2]: yep. Use ApplicationPolicy as ancestor

File: app/policies/permission_policy.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+class PermissionPolicy < RolePolicy
+end
```
Comment (dmitrymalllpr) [inline app/policies/permission_policy.rb:2]: Here!!! strange... Role is not Policy. Better EntityPolicy<BasePolicy DictionaryEntityPolicy <DictionaryPolicy ect

**Review by dmitrymalllpr — CHANGES_REQUESTED** [review]

File: app/policies/permission_policy.rb (hunk 3 above)
Comment (dmitrymalllpr) [reply app/policies/permission_policy.rb]: Use ApplicationPolicy as ancestor

**Review by KarinaKlevzhits — APPROVED** [review]

**Review by dmitrymalllpr — APPROVED** [review]

**Pull request merged**